.changeset/factory-review-verdict-wakes-work.md

```
@@ -0,0 +1,18 @@
+---
+'@mastra/factory': patch
+---
+
+Let a Factory review verdict reach the agent that authored the pull request.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)